src/components/FlowChart/FlowSelection.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Dropdown } from "react-bootstrap";
+
+export default function FlowSelection({
```

Comment:
why no named export?
if not typescript it would be important to use proptypes for new components.

Reply:
is `FlowSelection` not the name of this export?

Reply:
It doesn't have a name (:

`export { FlowSelection }`